packages/office-ui-fabric-react/src/components/OverflowSet/OverflowSet.tsx

```diff
@@ -32,6 +37,8 @@ export class OverflowSet extends BaseComponent<IOverflowSetProps, {}> implements
   }
 
   public render() {
+    this._unregisterPersistedKeytips();
```

Comment:
Do we need to unregisterPersistedKeytips on every render? Could we put this inside of either ComponentWillReceiveProps or ComponentWillUpdate and only unregister when needed?